app/src/main/java/com/github/swent/echo/compose/components/EventInfoSheet.kt

```diff
@@ -108,7 +110,14 @@ fun EventInfoSheet(
                 style = MaterialTheme.typography.bodyLarge
             )
 
-            Row(modifier = Modifier.align(Alignment.BottomCenter).padding(bottom = 70.dp)) {
+            val rowModifier =
```

Comment:
can be a one-liner : Modifier.align(Alignment.BottomCenter).padding(bottom = 70.dp, end = if(canModifyEvent) 140.dp else 0.dp)

Reply:
makes sense, didn't think about that